### PR TITLE
Fix fast glitch

### LINKED ISSUE #4442: Fast template header color is green and background white on startup
I'm on the current `main` branch of Panel. One thing that has been annoying me about the FastTemplates is that if you network is slow, your template is big or some javascript goes wrong you end up with an app where the header background is white and the color is green. At least for a split of a second.

This looks very unprofessional. I will try to fix.

You can see it in the upper right corner in the video below

https://user-images.githubusercontent.com/42288570/218275833-84184f42-c52e-469a-96ea-2813dea5842e.mp4

```python
import panel as pn
import pandas as pd

pn.extension(template="fast")

data = pd.DataFrame({"x": [1]*1000, "y": [2]*1000})

pn.panel(data).servable()
```

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
Closes #4442

### CODE REVIEW

File: panel/template/fast/theme.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ class FastStyle(param.Parameterized):
     accent_base_color = param.String(default="#0072B5")
     collapsed_icon = param.String(default=COLLAPSED_SVG_ICON)
     expanded_icon = param.String(default=EXPANDED_SVG_ICON)
-    color = param.String(default="#00aa41")
+    color = param.String(default="#2B2B2B")
```
Comment (MarcSkovMadsen) [inline panel/template/fast/theme.py:43]: This is where the grenn header color was coming from.

File: panel/template/fast/grid/fast_grid_template.html [hunk 2]
```diff
@@ -54,6 +54,8 @@
   }
   #header-design-provider {
     --neutral-foreground-rest: {{ style.header_color }};
+    background: {{ style.header_background }};
```
Comment (MarcSkovMadsen) [inline panel/template/fast/grid/fast_grid_template.html:57]: If we set these, we will see the header_background and header_color immediately. And not after all the `fast` web components have been initialized.

**Comment by philippjfr** [comment]
I'm trying to upgrade to latest fast-components version so will likely fix this separately but suspect these changes will still be needed.

**Comment by philippjfr** [comment]
Applied these fixes in https://github.com/holoviz/panel/pull/4326

**Pull request closed without merging**